src/atrule/keyframes.rs

```diff
@@ -0,0 +1,161 @@
+use codemap::{Span, Spanned};
+
+use peekmore::PeekMoreIterator;
+
+use super::parse::ruleset_eval;
+use crate::error::SassResult;
+use crate::scope::Scope;
+use crate::selector::Selector;
+use crate::utils::{devour_whitespace, eat_number};
+use crate::value::Number;
+use crate::{RuleSet, Stmt, Token};
+
+#[derive(Debug, Clone)]
+pub(crate) struct Keyframes {
+    pub super_selector: Selector,
+    pub name: String,
+    pub block_list: Vec<(Vec<Spanned<KeyframeSelector>>, Vec<Spanned<Stmt>>)>,
+}
+
+#[derive(Debug, Clone)]
+pub(crate) enum KeyframeSelector {
+    From,
+    To,
+    Percentage(Number),
```

Comment:
You could rename this variant to `Percent`

Reply:
Yes, I recall you mentioned this but I personally think `Percent` is better. Nevermind, I will change this.